### DISCUSSION TITLE
imagelist is not as user-friendly as it was in v3

### DISCUSSION THREAD

**Discussion by eduardomart** [bolt/core#1789] [Ideas / Feature requests]
In v3 we could drag a group of images and drop it in the `imagelist` field, boom, list of images ready. 
In v4 it has to be done one by one, which could just be done with a set/repeater, so not sure the `imagelist` field type achieves anything different, or why we would want it the way it currently is?

Bolt v4 RC 39

**Comment by bobdenotter** [comment]
Correct, this is an area that can do with improvement. Also, the Redactor and Article fields both have different selectors for images as well. We should make one interface for this, for the different areas. But, it’ll be a bit of work, thinking about it should work and to implement it properly.. Let’s add it to `4.1` for now.

**Comment by develth** [comment]
Made an suggest for it a few days ago:

https://bolt.kampsite.co/suggestions/2001e77b-e08e-403e-9bb9-1bf981ced7c8
